eZ/Publish/Core/REST/Client/Values/User/User.php

```diff
@@ -77,6 +77,21 @@ public function getFieldsByLanguage($languageCode = null)
         return $this->content->getFieldsByLanguage($languageCode);
     }
 
+    /**
+     * This method returns the field for a given field definition identifier and language.
+     *
+     * If not set the initialLanguage of the content version is used.
+     *
+     * @param string $fieldDefIdentifier
+     * @param null $languageCode
```

Comment:
that's probably something our ez crew friends should say, but i'd probably do another pr then to change only that.